### ISSUE TITLE
Would be great to see any possible errors/fails from Firewalla on mobile apps!

### ISSUE THREAD

**Issue by saormart** [firewalla/firewalla#1735]
I know that it may get's it's way to you guys from the back-end, but would be great if it's possible at least for advanced users/debug purpose to receive alerts and show any possible issues in details that the box goes through ! May it could be helpful also to help improve the device security!

**Comment by jerrchen** [comment]
Can you give an example?   Our problem is, over loading the customer with information can be a good and a bad things ...  The bad part is, it will drive support crazy for errors that's may sound serious. 

But, we are thinking of adding such interfaces to the web console, would that be useful?

**Comment by saormart** [comment]
@jerrchen, You're right about it will drive support crazy for regular users, but I was thinking for the advanced users that could also contribute with the project by analyzing the errors and see what's going on. At the begin I was thinking in something to don't use the back end servers you guys have to don't affect any performance... something like to access the box local error logs (if it have some enabled of course).
So, adding also to the web interface would work !
I don't know how you guys handle all your costumers with tech support, but I can tell you guys are doing a great job! Keep it up !

**Comment by jumson** [comment]
@jerrchen Supporting that on web console would be great!
I concure with @saormart, maybe an "advanced" feature to simply dump error logs for sending via email or viewing in browser or something. 
I'm also interesed to see access logs for the firewalla, and for the VPN.
I guess there would need to be an ability to enable logging and to set the size limit / duration kept, and maybe a option to export/backup logs after size/duration period.
